package.json

```diff
@@ -31,7 +31,7 @@
     "test": "npm run clean && npm run lint && npm run spec"
   },
   "dependencies": {
-    "asar": "^0.13.0",
+    "asar": "^0.14.0",
```

Comment:
I still disagree with updating dependencies in unrelated PRs.

Reply:
Ok, from now on I won't include updating dependencies on PRs, unless it's a dedicated PR for it